<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infoForm</class>
 <widget class="QWidget" name="infoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="info_communication">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>58</y>
     <width>174</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="info_internet">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>222</y>
     <width>174</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="info_settings">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>58</y>
     <width>174</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="info_entertainment">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>222</y>
     <width>174</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="info_info">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>233</width>
     <height>236</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
